src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.ts

```diff
@@ -283,6 +290,12 @@ export class ServiceFormComponent extends CdForm implements OnInit {
       realm_name: [null],
       zonegroup_name: [null],
       zone_name: [null],
+      qat: new CdFormGroup(
+        this.rgwQatList.reduce((acc: object, e) => {
+          acc[e] = new UntypedFormControl(false);
+          return acc;
+        }, {})
+      ),
```

Comment:
for clarity, why is this boolean? I believe what we have is a dropdown so this shouldnt be a boolean controller is it?

and I think its more easier to understand if we defined all the controls here. We can expect this to grow but with form controls that behaves differently (checkboxes, textboxes...)

```suggestion
      qat: new CdFormGroup({
         compression: new UntypedFormControl(null)
      ),
```